### DISCUSSION TITLE
テスト：基礎1セミナーについて

### DISCUSSION THREAD

**Discussion by N-YOKOI** [toppers/users-forum#61] [TOPPERS Seminar]
テストとして質問を投稿します。

**Comment by Ryohsuke-Takeuchi** [comment]
contributorにならなくても、回答できますか？
できるならならなくても良いですが、通知がないと回答を見落とすかもしれません。

**Comment by Ryohsuke-Takeuchi** [comment]
基礎１セミナーは、ベアメタルの組込みソフトウェアを開発する教材です。ベアメタルでは、スタートアップルーチンやデバイスを制御するプログラムを自分で書いたり、不具合の対応を作成者が行う必要があり、組込みの多様な知識が開発では必要となります。基礎１では、GPIO、タイマー、シリアル通信などのプログラムを、受講者が作るコンテンツとなってます。使用するボードはSTM32F401-nucleo64ボードと、LEDやプッシュスイッチやDIP-SWをプロトタイプシールドに実装したものを使用します。通信教材では、プロトタイプシールドへの部品実装は、受講者が行う必要があります。
